docs/tutorials/testing_with_control_center.rst

```diff
@@ -108,98 +101,111 @@ though without the colored highlighting.
+in the top left and the Information section will be populated with information about the connectd module. The image
+below shows an example of what the Control Center will look like when connecting to a Vertiq 8108. If the module does not
 yet have a bootloader or upgrader, N/A is filled into the respective version.
 
-.. warning:: Before proceeding any further, check that the firmware style indicates this motor is using speed firmware.
+.. warning:: Before proceeding any further, check that the firmware style indicates this module is using speed firmware.
     This tutorial is only intended for speed firmware, and not for position firmware.
 
-.. figure:: ../_static/tutorial_images/control_center_tutorial/control_center_connected_.JPG
+.. figure:: ../_static/tutorial_images/control_center_tutorial/control_center_connected.PNG
     :align: center
     :alt: Control Center Connected to 8108
 
     Control Center Connected to Vertiq 8108
 
 Updating Firmware
 =================
-For Control Center Versions Below 1.3.0: 
-If you do need to update the motor firmware, you can do it through the Control Center. First download the firmware binary you want to update to, and then open the Firmware tab. 
-Click on “Select Firmware Binary”, and select the binary you downloaded. Make sure you are connected to the motor, and then hit “Flash”. 
-The motor should disconnect, and a progress bar at the bottom of the Control Center should start filling. When the progress bar is full, the flash is complete. 
-The motor should restart and play its 5 beep startup song. An example of the Control Center after a successful flash is shown below.
-
-For Versions Above 1.3.0:
-If you do need to update the motor firmware, you can do it through the Control Center. First, download the most recent firmware from vertiq.co, either .bin or .zip (if available).
-Click on "Select Firmware (".bin" or ".zip")", and select the file or archive you downloaded. Make sure you are connected to the motor. 
+If you need to update the module firmware, you can do it through the Control Center. Older versions of Control Center lack some of the firmware update features of more 
+recent updates. The sections below cover the process for older and more recent versions of Control Center.
+
+Control Center Versions Below 1.3.0
+************************************
+First download the firmware binary you want to update to from the `Vertiq website <https://www.vertiq.co/>`_, and then open the Firmware tab. 
+Click on “Select Firmware Binary”, and select the binary you downloaded. Make sure you are connected to the module, and then hit “Flash”. 
+The module should disconnect, and a progress bar at the bottom of the Control Center should start filling. When the progress bar is full, the flash is complete. 
+The module should restart and play its 5 beep startup song. An example of the Control Center after a successful flash is shown below.
```

Comment:
where is this picture? did you maybe mean during a flash?

Reply:
This is an old reference to a picture that no longer exists, removed this sentence